src/controllers/users.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const {
+  getAll: getAllUsers,
+  getOne: getOneUser,
+  create: createUser,
+  remove: removeUser,
+  update: updateUser,
+  normalize,
+} = require('../services/users');
+
+const getAll = async(req, res) => {
+  const users = await getAllUsers();
+
+  res.send(users.map(normalize));
+};
+
+const getOne = async(req, res) => {
+  const { userId } = req.params;
+
+  if (isNaN(+userId)) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = await getOneUser(userId);
+
+  foundUser ? res.send(normalize(foundUser)) : res.sendStatus(404);
+};
```

Comment:
You should send status code 200 on success

Reply:
But we have this status by default 🤔